### PR TITLE
Conv transpose op

### PR DESCRIPTION
**Pull request by kahmed10**

### CODE REVIEW

File: src/onnx/onnx.cpp [hunk 1]
```diff
@@ -277,6 +278,25 @@ struct onnx_parser
         });
     }
 
+    template <class T>
+    std::vector<int64_t> to_int64_vector(std::vector<T> input_vector)
```
Comment (pfultz2) [inline src/onnx/onnx.cpp]: This should be passed as const ref.

File: src/onnx/onnx.cpp [hunk 2]
```diff
@@ -277,6 +278,25 @@ struct onnx_parser
...
+    {
+        std::vector<int64_t> output_vector(input_vector.begin(), input_vector.end());
+        return output_vector;
+    }
+
+    instruction_ref
+    add_bias(std::vector<instruction_ref> args, instruction_ref curr_ins, uint64_t axis)
```
Comment (pfultz2) [inline src/onnx/onnx.cpp]: The vector should be passed as const ref.

Comment (pfultz2) [reply src/onnx/onnx.cpp]: Also does this `add_bias` function work for convolution as well?

Comment (kahmed10) [reply src/onnx/onnx.cpp]: > Also does this `add_bias` function work for convolution as well?

yes it does. I replaced it in the parse_conv function

File: src/onnx/onnx.cpp (hunk 1 above)
Comment (kahmed10) [reply src/onnx/onnx.cpp]: fixed

**Pull request merged**